<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1350</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1350</width>
    <height>650</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Leelawadee</family>
    <pointsize>16</pointsize>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>TRADING</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="start_date">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferNumbers</set>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::PlusMinus</enum>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="keyboardTracking">
     <bool>false</bool>
    </property>
    <property name="minimumDate">
     <date>
      <year>2021</year>
      <month>3</month>
      <day>15</day>
     </date>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="stop_date">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="minimumDate">
     <date>
      <year>2021</year>
      <month>3</month>
      <day>15</day>
     </date>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="twitter_lb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Twitter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="twitter_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Enter Twitter's Keyword</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="news_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Enter News' Keyword</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="news_lb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>News</string>
    </property>
   </widget>
   <widget class="QLabel" name="dad_lb">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;-&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="stop_date_lb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#7e7e7e;&quot;&gt;To&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="start_date_lb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#7e7e7e;&quot;&gt;From&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="group_new">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>511</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>News</string>
    </property>
    <widget class="mpl_related_news" name="news_related_widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="mpl_sentiment_news" name="news_sentiment_widget" native="true">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>241</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_trendy">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>251</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Trendy</string>
    </property>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>231</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="search_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SEARCH</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="group_stock">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>1041</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>Stock</string>
    </property>
    <widget class="QLineEdit" name="stock_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Enter Stock's SYMBOLS</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="submit_btn">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>SUBMIT</string>
     </property>
    </widget>
    <widget class="mpl_stock" name="stock_widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>1021</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_twitter">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>511</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Twitter</string>
    </property>
    <widget class="mpl_related_twitter" name="twitter_related_widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="mpl_sentiment_twitter" name="twitter_sentiment_widget" native="true">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>241</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <zorder>start_date</zorder>
   <zorder>stop_date</zorder>
   <zorder>twitter_lb</zorder>
   <zorder>twitter_input</zorder>
   <zorder>news_lb</zorder>
   <zorder>dad_lb</zorder>
   <zorder>stop_date_lb</zorder>
   <zorder>start_date_lb</zorder>
   <zorder>group_new</zorder>
   <zorder>group_trendy</zorder>
   <zorder>search_btn</zorder>
   <zorder>group_stock</zorder>
   <zorder>group_twitter</zorder>
   <zorder>news_input</zorder>
   <zorder>clear_btn</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mpl_related_twitter</class>
   <extends>QWidget</extends>
   <header>mpl_related_twitter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mpl_sentiment_twitter</class>
   <extends>QWidget</extends>
   <header>mpl_sentiment_twitter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mpl_related_news</class>
   <extends>QWidget</extends>
   <header>mpl_related_news.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mpl_sentiment_news</class>
   <extends>QWidget</extends>
   <header>mpl_sentiment_news.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mpl_stock</class>
   <extends>QWidget</extends>
   <header>mpl_stock.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
